packages/components/src/components/collapsible/collapsible.tsx

```diff
@@ -26,11 +27,13 @@ const Collapsible = ({
     onClick,
     title,
     handle_button,
+    switch_off_functionality = false,
```

Comment:
I rename this props to make it clear what it's doing. Whenever `switch_off_functionality = true`, the I'm hiding handle button and remove functionality of expanding/collapsing. This can't be done only with CSS

Reply:
what about `is_noninteractive`? :)

Reply:
or turn_off_toggle

Reply:
or diable/enable toggle